### ISSUE TITLE
Does not receive messages from second WhatsApp cloud account

### ISSUE THREAD

**Issue by jeanmarcosdarosa** [chatwoot/chatwoot#8845]
### Describe the bug

Hello, I have a WhatsApp Cloud account that works perfectly, but when I add a second account, the messages received are not displayed on Chatwoot. As I saw in some discussions here on github, some people managed to solve it by deleting the phone and adding it again to the Facebook app, I tried the same and it didn't solve it, initially I had added the new number to the same Facebook app, and I believed that the problem was just there is a webhook, but I carried out the process indicated in the FAQ, of using just one of the webhooks provided by chatwoot, it didn't solve the problem, so I created a new app on Facebook, and re-registered the inbox on chatwoot, the same problem continues.

The interesting thing is that Facebook sends the webhook correctly when I receive a message, but Chatwoot doesn't register it.
Example of the webhook log that Facebook sends:

`{"object"=>"whatsapp_business_account", "entry"=>[{"id"=>"237542506102556", "changes"=>[{"value"=>{"messaging_product"=>"whatsapp", "metadata"=>{"display_phone_number"=>"555496634363", "phone_number_id"=>"174639642410376"}, "contacts"=>[{"profile"=>{"name"=>"Jean"}, "wa_id"=>"555499261407"}], "messages"=>[{"from"=>"555499261407", "id"=>"wamid.HBgMNTU1NDk5MjYxNDA3FQIAEhggRDc0Rjc5MzMwNDRBRTI4QjhCMUM0OTRBNzA4OTU1MkMA", "timestamp"=>"1706820155", "text"=>{"body"=>"Ola"}, "type"=>"text"}]}, "field"=>"messages"}]}], "controller"=>"webhooks/whatsapp", "action"=>"process_payload", "phone_number"=>"+5554996634363", "whatsapp"=>{"object"=>"whatsapp_business_account", "entry"=>[{"id"=>"237542506102556", "changes"=>[{"value"=>{"messaging_product"=>"whatsapp", "metadata"=>{"display_phone_number"=>"555496634363", "phone_number_id"=>"174639642410376"}, "contacts"=>[{"profile"=>{"name"=>"Jean"}, "wa_id"=>"555499261407"}], "messages"=>[{"from"=>"555499261407", "id"=>"wamid.HBgMNTU1NDk5MjYxNDA3FQIAEhggRDc0Rjc5MzMwNDRBRTI4QjhCMUM0OTRBNzA4OTU1MkMA", "timestamp"=>"1706820155", "text"=>{"body"=>"Ola"}, "type"=>"text"}]}, "field"=>"messages"}]}]}}`

### To Reproduce

Chatwoot version: v3.5.2
Build 608592db0c11db142c9a281d1d34b715ad33d6c0

### Environment

Docker

### Cloud Provider

Other [please specify in the description]

### Platform

Mobile

### Operating system

Linux

**Comment by sojan-official** [comment]
@jeanmarcosdarosa did you create the new inbox with the correct phone number ?

**Comment by jeanmarcosdarosa** [comment]
Thank you so, so, so much! Your response was direct and clear. In my case, the phone number is from Brazil, and here we use the ninth digit, which is a standard that Brazil has adopted in recent years. When I created the phone number on WhatsApp Business, I provided the ninth digit, but I didn't realize that the input form on Facebook removed the ninth digit, creating the account with one digit less. I recreated the input box on Chatwoot without the ninth digit, and that solved my problem. Thank you very much!
